subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -1052,6 +1052,10 @@ config BT_MAYFLY_YIELD_AFTER_CALL
 	  If set to 'n', all pending mayflies for callee are executed before
 	  yielding
 
+config BT_TICKER_CNTR_FREERUNNING
```

Comment:
Not sure what it is about the name, but `FREERUNNING` hurts my eyes :D Suggest `FREE_RUNNING`